samples/kubernetes/end2end/docs/06-grafana.md

```diff
@@ -9,9 +9,10 @@ Create the Grafana administrative credentials.
 ```
 kubectl --namespace monitoring create secret generic grafana-secret --from-literal=username=admin --from-literal=password=12345678
 ```
-Install the Grafana chart.
+Install the Grafana chart, in this sample we will use Helm chart version 6.38.6.
```

Comment:
Once again, I believe this comma should be a semicolon or period.

Reply:
Correct; re-write it however you did in the previous instance.

Reply:
fixed